cli/popper/commands/cmd_run.py

```diff
@@ -51,13 +54,11 @@ def cli(ctx, pipeline, timeout, skip):
                 status = run_pipeline(project_root, pipes[pipe], timeout, skip)
 
                 if status == 'FAIL':
+                    pu.fail("Failed to execute pipeline")
```

Comment:
This was actually at the right indentation level (on line 58 on the previous commit), since we want to fail the case when a pipeline failed on line 45